.coderabbit.yaml

```
@@ -0,0 +1,15 @@
+# .coderabbit.yaml
+language: "ko-KR"
+early_access: false
+reviews:
+  profile: "chill" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)